db/default/users.rb

```diff
@@ -23,20 +23,20 @@ def prompt_for_admin_email
     email = ENV['ADMIN_EMAIL'].dup
     say "Admin User #{email}"
   else
-    email = ask('Email [spree@example.com]: ') do |q|
+    email = ask('Email [ofn@example.com]: ') do |q|
       q.echo = true
       q.whitespace = :strip
     end
-    email = 'spree@example.com' if email.blank?
+    email = 'ofn@example.com' if email.blank?
   end
 
   email
 end
 
 def create_admin_user
   if ENV.fetch("AUTO_ACCEPT", true)
-    password = ENV.fetch("ADMIN_PASSWORD", "spree123").dup
-    email = ENV.fetch("ADMIN_EMAIL", "spree@example.com").dup
+    password = ENV.fetch("ADMIN_PASSWORD", "ofn123").dup
+    email = ENV.fetch("ADMIN_EMAIL", "ofn@example.com").dup
```

Comment:
these are the values specified as ENV vars in the docker-compose.yml. Could you please update that too?

Reply:
sorry I dont understand what do you mean.
the values in docker-compose are already ofn@example.com ofn123 credits to @pacodelaluna btw

Reply:
I mean that since the env vars defined in the docker-compose.yml specify the values that are now the defaults, there's no need to provide them. Without them, password and email will be the same.

Reply:
ah, ok!
done :+1: 
and tested locally, it works.